guides/installation/overview.md

```diff
@@ -74,7 +74,7 @@ PHP
 SQL
 
 * MySQL 5.7.21 or higher
-  * MySQL 8.0.20 is not compatible
+  * MySQL 8.0.20 is not compatible (only this specific version, 8.0.23 for example is working)
```

Comment:
I'm not entirely sure why we would need a change here to be honest, as it literally only mentions a single version and not MySQL as a whole.
However, I guess I'd rather have it like this. What do you think?

```suggestion
  * Only MySQL 8.0.20 in specific is not compatible
```

Reply:
Fine with that :+1: 
I heared already 5 peoples that thinks that mysql 8 in total is not supported.
So I am totaly sure this change is needed. Thx.